### PR TITLE
Add Mario & Sonic at the London 2012 Olympic Games

### PR DESCRIPTION
**Pull request by robzombie91**
Adds Mario & Sonic at the London 2012 Olympic Games.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/mario-sonic-at-the-london-2012-olympic-games/game.dat [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+[[releases]]
+title = "0004000000032500"
+region = "jpn"
+release_date = "2012-03-01"
+
+[[releases]]
+title = "0004000000054800"
+region = "usa"
+release_date = "2013-06-20"
```
Comment (Leo626) [inline games/mario-sonic-at-the-london-2012-olympic-games/game.dat]: What's with the duplicate usa release?

Comment (robzombie91) [reply games/mario-sonic-at-the-london-2012-olympic-games/game.dat]: ehsop

File: games/mario-sonic-at-the-london-2012-olympic-games/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
+title = "Mario & Sonic at the London 2012 Olympic Games"
+description = "Mario & Sonic at the London 2012 Olympic Games[a] is a sports and party game developed by Sega Japan. As with the previous two Mario & Sonic titles, it was published by Nintendo in Japan and by Sega in all other regions. The game is officially licensed by the International Olympic Committee (IOC) through exclusive licensee International Sports Multimedia. It is the third installment in the Mario & Sonic series after the commercial success of its predecessors."
```
Comment (Leo626) [inline games/mario-sonic-at-the-london-2012-olympic-games/game.dat]: Remove the `[a]`.

File: games/mario-sonic-at-the-london-2012-olympic-games/game.dat [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+region = "eur"
+release_date = "2012-02-09"
+
+[[releases]]
+title = "0004000000032500"
+region = "jpn"
+release_date = "2012-03-01"
+
+[[releases]]
+title="0004000000053500"
```
Comment (Hexagon12) [inline games/mario-sonic-at-the-london-2012-olympic-games/game.dat]: Spacing. `title = "..."`

**Comment by Leo626** [comment]
Nit: At least one screenshot with gameplay.

**Pull request merged**